### PR TITLE
[Core] clean includess

### PR DESCRIPTION
**Pull request by philbucher**
just some minor includes-cleaning

### CODE REVIEW

File: kratos/includes/element.h [hunk 1]
```diff
@@ -10,30 +10,21 @@
...
 // Project includes
-#include "includes/define.h"
-#include "includes/node.h"
-#include "geometries/geometry.h"
 #include "includes/properties.h"
 #include "includes/process_info.h"
 #include "includes/geometrical_object.h"
-#include "utilities/indexed_object.h"
 #include "containers/flags.h"
-#include "containers/weak_pointer_vector.h"
```
Comment (RiccardoRossi) [inline kratos/includes/element.h:34]: I think that this is needed to correctly define the NEIGHBOUR_CONDITIONS

Comment (philbucher) [reply kratos/includes/element.h:34]: But NEIGHBOR_CONDITIONS is a variable right? Shouldn’t this then include it?
Compiles fine in the CI

Comment (RiccardoRossi) [reply kratos/includes/element.h:34]: @philbucher the problem is the definition of that variable. right now it is done in the header.

examples of this are NEIGHBOUR_NODES, NEIGHBOUR_ELEMENTS, NEIGHBOUR_CONDITIONS defined in node.h, element.h, condition.h (all of these are WeakPointerVectors)

Comment (philbucher) [reply kratos/includes/element.h:34]: hm I still don't get it (also it is not included in `Condition` btw) but you seems sure so I brought it back

Comment (RiccardoRossi) [reply kratos/includes/element.h:34]: https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/includes/condition.h#L1257

am i getting you wrongly?

**Review by RiccardoRossi — DISMISSED** [review]

File: kratos/includes/element.h (hunk 1 above)
Comment (philbucher) [reply kratos/includes/element.h:34]: Ah there it is, didn't see this, sorry
I will add the include also to `condition.h` then

**Comment by philbucher** [comment]
can you reapprove pls @RiccardoRossi ?

**Review by RiccardoRossi — APPROVED** [review]

**Pull request merged**